classes/class-wpcom-liveblog-wp-cli.php

```diff
@@ -11,6 +13,105 @@ public function readme_for_github() {
 		$readme = $this->markdownify_headings( $readme );
 		echo $readme;
 	}
+	
+	/**
+	 * Fix wp_commentmeta table so archived liveblog posts comments display properly.
+	 *
+	 * @subcommand fix-archive
+	*/
+	public function fix_archive() {
```

Comment:
This command is pretty quiet :) We should get some more output so that it's clear to the user what the command is doing. Progress might be useful, too.

Reply:
Also, a dry-run option would be best.

Reply:
@philipjohn all the other tweaks are completed Ill tackle this one tomorrow morning first thing .